### PR TITLE
Update to latest native sdks

### PR DESCRIPTION
**Pull request by asutula**
Updated `Profile.setAvatar` and `Files.list` methods

Trying to work through some problems updating the generated JS types, but wanted to get this out for PR asap.

### CODE REVIEW

File: src/files.ts [hunk 1]
```diff
@@ -84,11 +84,11 @@ export async function addByTarget(target: string, threadId: string, caption?: st
 /**
  * List all files or files in a known Thread.
  * ```typescript
- * Textile.files.list(offset, limit);
+ * Textile.files.list(threadId, offset, limit);
  * ```
  */
-export async function list(offset: string, limit: number, threadId?: string): Promise<IFilesList> {
-  const result = await FilesBridge.list(offset, limit, threadId)
+export async function list(threadId: string, offset: string, limit: number): Promise<IFilesList> {
```
Comment (asutula) [inline src/files.ts]: Snuck your change in here @andrewxhill

**Comment by andrewxhill** [comment]
if you don't mind, hit `yarn docs` on or after this PR so we can get updated live docs

File: README.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ The tsdoc-based auto-generated documentation can be found at https://textileio.g
 <!-- ⛔️ AUTO-GENERATED-CONTENT:START (CONTRIBUTORS) -->
 | **Commits** | **Contributor** |  
 | --- | --- |  
-| 66 | [andrewxhill](https://github.com/andrewxhill) |  
-| 18 | [sanderpick](https://github.com/sanderpick) |  
-| 17 | [asutula](https://github.com/asutula) |  
+| 69 | [andrewxhill](https://github.com/andrewxhill) |  
+| 24 | [asutula](https://github.com/asutula) |
```
Comment (asutula) [inline README.md]: Sweet, overtook @sanderpick

File: docs/classes/eventsubscription.html [hunk 3]
```diff
@@ -0,0 +1,254 @@
+<!doctype html>
```
Comment (asutula) [inline docs/classes/eventsubscription.html]: No idea why this file was created.

**Pull request merged**